source/overloading.tex

```diff
@@ -1912,9 +1909,7 @@
 \indextext{overloading!resolution!implicit conversions and|(}
 
 \pnum
-An
-\term{implicit conversion sequence}
-\indextext{sequence!implicit conversion}%
+An \defnx{implicit conversion sequence}{sequence!implicit conversion}
```

Comment:
Yuck. Can we index this as "conversion sequence!implicit" rather than "sequence!implicit conversion"? Also, a \see from "implicit conversion sequence" would be really useful.

Reply:
Fixed.